DcCore/DcCore/Extensions/String+Extensions.swift

```diff
@@ -2,8 +2,12 @@ import Foundation
 import UIKit
 
 public extension String {
-    
-	static func localized(_ stringID: String) -> String {
+
+    static func markAsExternal(_ string: String) -> String {
+        return string + " ↗"
```

Comment:
tested rtl, that looks good as well - it was anyway unclear if the arrow should become first or last, and even more with mixed directions, both seems fine - until we get some different hints